.github/workflows/build-test-publish.yml

```diff
@@ -247,10 +247,11 @@ jobs:
             python -m pip install itkwidgets
             # Issues with 5.7.0
             # https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/issues/407
-            python -m pip install traitlets==5.6.0
+            python -m pip install traitlets==5.6.0 pytest nbmake
 
       - name: Test notebooks
-        uses: treebeardtech/nbmake-action@v0.2.1
+        run: |
+          pytest --nbmake --nbmake-timeout=30000
```

Comment:
Thanks @jhlegarreta . I think we need to specify the path the notebook here, correct?